### ISSUE TITLE
banner image acts strange

### ISSUE THREAD

**Issue by amigax** [FidelLimited/fidel-android#7]
Banner image wont appear until you have entered a cc number, then it slides the activity up and only slides back down to see banner once you tick the checkbox

**Comment by ccorneliu** [comment]
@amigax Yes, we're sorry. We are aware of this issue. We'll work on improving the banner image behavior.
